src/dxvk/dxvk_presenter.cpp

```diff
@@ -151,6 +163,8 @@ namespace dxvk {
 
 
   VkResult Presenter::recreateSwapChain(const PresenterDesc& desc) {
+    std::lock_guard<dxvk::mutex> lock(m_lowLatencyMutex);
```

Comment:
`recreateSurface` can also result in the swapchain getting destroyed. I think we currently lack locking there (and this might be a tricky one since `recreateSurface` will keep `m_swapchain` NULL until the next call to `recreateSwapchain`).

Reply:
Updated this to only take the lock when destroying the swapchain. Whenever dxvk calls one of the other LL2 entry points it holds the low latency lock, and confirms the swapchain handle is valid. This approach should prevent any usage of the destroyed swapchain handle.